### PR TITLE
[OV JS][DOCS] Edits in JS API docs

### PR DESCRIPTION
**Pull request by vishniakov-nikolai**
### Details:
 - Add Node.js API contributing guide
 - Edits into `openvino-node` readme
 - Minor documentation fixes

### Tickets:
 - 149666

### CODE REVIEW

**Review by almilosz — APPROVED** [review]
Content looks good to me ;) 
Let's wait for the documentation team ;)

File: src/bindings/js/node/README.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
 
-Use OpenVINO JavaScript API for your Node.js application.
+Execute deep learning models in Node.js applications with ease using OpenVINO™.
 
-## Usage
+## Introduction
+
+OpenVINO™ is an open-source toolkit designed for high-performance deep learning inference.
+Node.js API provides bindings to subset APIs from OpenVINO Runtime.
+These Node.js bindings allow JavaScript developers to use the power of OpenVINO™ in their applications.
```
Comment (sgolebiewski-intel) [inline src/bindings/js/node/README.md:9]:
```suggestion
The Node.js bindings enable JavaScript developers to use the capabilities of OpenVINO in their applications.
```

File: src/bindings/js/node/README.md [hunk 2]
```diff
@@ -14,15 +20,21 @@ Use the **openvino-node** package:
 const { addon: ov } = require('openvino-node');
 ```
 
+Full `addon` API you can find at documentation [page](https://docs.openvino.ai/2024/api/nodejs_api/addon.html).
```
Comment (sgolebiewski-intel) [inline src/bindings/js/node/README.md:23]:
```suggestion
Refer to the complete description of the `addon` API in the [documentation](https://docs.openvino.ai/2024/api/nodejs_api/addon.html).
```

File: src/bindings/js/node/README.md [hunk 3]
```diff
@@ -14,15 +20,21 @@ Use the **openvino-node** package:
...
+
+See more samples in [samples section](https://github.com/openvinotoolkit/openvino/blob/master/samples/js/node/README.md).
```
Comment (sgolebiewski-intel) [inline src/bindings/js/node/README.md:25]:
```suggestion
See the [samples](https://github.com/openvinotoolkit/openvino/blob/master/samples/js/node/README.md) for more details on how to use it.
```

File: src/bindings/js/node/README.md [hunk 4]
```diff
@@ -14,15 +20,21 @@ Use the **openvino-node** package:
...
 **Desktop development with C++** component from
 [Build Tools for Visual Studio](https://aka.ms/vs/17/release/vs_BuildTools.exe) is installed.
 
-## Build From Sources
+## Supported Platforms
 
-For more details, refer to the [OpenVINO™ JavaScript API Developer Documentation](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/js/docs/README.md#openvino-node-package-developer-documentation)
+- Windows x86
+- Linux x86/arm
+- MacOS x86/arm
```
Comment (sgolebiewski-intel) [inline src/bindings/js/node/README.md:37]:
```suggestion
- Linux x86/ARM
- MacOS x86/ARM
```

File: src/bindings/js/node/README.md [hunk 5]
```diff
@@ -31,11 +43,19 @@ For more details, refer to the [OpenVINO™ JavaScript API Developer Documentati
...
 
 - [hello-classification-sample](https://codesandbox.io/p/devbox/openvino-node-hello-classification-sample-djl893)
 
+## Build From Sources
+
+For more details, refer to the [OpenVINO™ JavaScript API Developer Documentation](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/js/docs/README.md#openvino-node-package-developer-documentation)
+
+## Contributing
+
+We welcome contributions! Please read our [Contributing Guide](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/js/node/CONTRIBUTING.md) to learn how you can get involved.
```
Comment (sgolebiewski-intel) [inline src/bindings/js/node/README.md:57]:
```suggestion
Contributions are always welcome! Read the [Contribution Guide](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/js/node/CONTRIBUTING.md) to learn how you can get involved.
```

File: src/bindings/js/node/README.md [hunk 6]
```diff
@@ -31,11 +43,19 @@ For more details, refer to the [OpenVINO™ JavaScript API Developer Documentati
 
 ## Live Sample
 
-You can run this sample in the browser; no installation is required.
+You can run this sample in the browser, no installation is required.
```
Comment (sgolebiewski-intel) [inline src/bindings/js/node/README.md:46]:
```suggestion
You can run the following sample in the browser, no installation is required.
```

File: src/bindings/js/node/CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+- Decide what you want to change
+- Create your fork of the openvino repository
+- Implement changes, align code style, run tests
+- Keep changes in the branch with a clear name
+- Create a Pull Request, clearly describe what has been changed and why
+- Go through Code Review
+- Be merged, you are awesome!
+
+Detailed explanation below:
```
[the remark this answers is not in the document]
Comment (almilosz) [reply src/bindings/js/node/CONTRIBUTING.md:19]: I think this order is more clear :)

**Pull request merged**